tests/acceptance/init.spec.js

```diff
@@ -53,7 +53,7 @@ describe('Acceptance: ng init', function() {
         });
 
         expected.forEach(function (file, index) {
-            expected[index] = file.replace(/__name__/g, 'angular-cli');
+            expected[index] = file.replace(/__name__/g, 'tmp');
```

Comment:
As far as I can tell, the API hasn't changed per se. It just so happens that the init command on `ember-cli` doesn't use any variables (like `__name__`) on it's blueprints for file paths, whereas we do.

So in line 119 we're using an explicit name (where before we weren't) and here in the test we're manually replacing the `__name__` template string.

It just seems that the test before defaulted to `angular-cli` when no name was given.